app/templates/modules/footer.njk

```diff
@@ -0,0 +1,13 @@
+<footer>
+  <p>
+    <a href="https://lil.law.harvard.edu">
+      <img src="/static/lil.svg" alt="Harvard Library Innovation Lab - Logo">
+    </a>
+  </p>
+  
+  <p>This site is an experiment of the <a href="https://lil.law.harvard.edu">Harvard Library Innovation Lab</a>.</p>
+
+  <p><a href="https://twitter.com/harvardlil">@harvardlil</a> on Twitter - <a href="https://law.us3.list-manage.com/subscribe?u=4290964398813d739f2398db0&amp;id=e097736c6f">Subscribe to our Newsletter</a>.</p>
```

Comment:
Want to replace this with the short link?

Reply:
Good call :) !